### DISCUSSION TITLE
How do I download full faker.js and faker.min.js?

### DISCUSSION THREAD

**Discussion by wojciechrak** [faker-js/faker#974] [Q&A] [answered]
Where can I download single-file faker.js and faker.min.js?
For version 5.5.3 I can see the files here: https://unpkg.com/browse/@faker-js/faker@5.5.3/dist/
But for >=6.0.0 there are only cjs and esm.

**Comment by Shinigami92** [comment, marked as answer]
~~You cannot do that anymore~~
We have a fully new bundling process
If you need to bundle everything into one single file you need to look for a custom solution 

EDIT: See below for suggestions

**Reply by AStarCale** [reply]
I'm an engineer, I can't get any of these faker.js tutorials to work. They're all broken. You took out the faker.min.js so I can't just import the script. the "import { faker } from 'https://esm.sh/@faker-js/faker';" line does not work, I've been here for hours and I can't even get one ting to display. The entire point of the way they made JavaScript was to NOT BREAK THE INTERNET when the new version. You broke the internet.

**Reply by xDivisionByZerox** [reply]
"CookingWithCale"? I don't no what you cooking but it ain't it. It literally takes  11 lines of html to get the MVP [from our docs](https://fakerjs.dev/guide/usage.html#browser) to run.

```html
<body>
  <script type="module">
  import { faker } from 'https://esm.sh/@faker-js/faker';

  const randomName = faker.person.fullName();
  const randomEmail = faker.internet.email();

  console.log({ randomName, randomEmail })
  </script>
</body>
```

Put this exact content in a "file_name.html" file, open it with your browser and see the log in the console.

**Reply by ST-DDT** [reply]
The following script also works in the browser console:

````ts
const { faker } = await import("https://cdn.jsdelivr.net/npm/@faker-js/faker/+esm");
faker.string.alpha(5)
````

**Reply by ST-DDT** [reply]
I created a PR to include the browser console example in our usage guide:

- https://github.com/faker-js/faker/pull/2919
